tests/Lithuania/EasterDayTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * This file is part of the Yasumi package.
+ *
+ * Copyright (c) 2015 - 2017 AzuyaLabs
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @author Sacha Telgenhof <stelgenhof@gmail.com>
+ */
+
+namespace Yasumi\tests\Lithuania;
+
+use Yasumi\Holiday;
+use Yasumi\tests\YasumiTestCaseInterface;
+
+/**
+ * Class containing tests for Easter day in Lithuania.
+ *
+ * @author Gedas Lukošius <gedas@lukosius.me>
+ */
+class EasterDayTest extends LithuaniaBaseTestCase implements YasumiTestCaseInterface
+{
+    /**
+     * The name of the holiday to be tested
+     */
+    const HOLIDAY = 'easter';
+
+    /**
+     * Test defined holiday in the test
+     */
+    public function testHoliday()
+    {
+        $year = 2017;
```

Comment:
It's recommended not to hardcode a unit test for certain year. (Please have a look at some of the other countries' unit tests).